reflect.go

```diff
@@ -0,0 +1,456 @@
+package main
+
+import (
+	"fmt"
+	"go/types"
+	"path/filepath"
+
+	"golang.org/x/exp/maps"
+	"golang.org/x/exp/slices"
+	"golang.org/x/tools/go/ssa"
+)
+
+var checkedAPIs = make(map[string]bool)
+
+// Record all instances of reflection use, and don't obfuscate types which are used in reflection.
+func (tf *transformer) recordReflection(ssaPkg *ssa.Package) {
+	if reflectSkipPkg[ssaPkg.Pkg.Name()] {
```

Comment:
you likely want to skip packages based on path, not name - otherwise `text/template` and `html/template` will both just be `template`